requirements.txt

```diff
@@ -1351,13 +1293,10 @@ sniffio==1.3.1 \
 social-auth-app-django==5.4.1 \
```

Comment:
I think with the pining of social-auth-app-django==5.4.0 this won’t be an issue, but wanted to call it out just in case